contracts/JetStakingV2.sol

```diff
@@ -21,7 +21,7 @@ import "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeab
  *      This contract is AdminControlled which has a tremendous power. However
  *      hopfully it be governed by a community wallet.
  */
-contract JetStakingV1 is AdminControlled {
+contract JetStakingV2 is AdminControlled {
     using SafeERC20Upgradeable for IERC20Upgradeable;
     bytes32 public constant AIRDROP_ROLE = keccak256("AIRDROP_ROLE");
     bytes32 public constant CLAIM_ROLE = keccak256("CLAIM_ROLE");
```

Comment:
If it is an upgradable contract then constructor doesn't make any sense here 🤔

Reply:
This constructor is (the implementation constructor) which initializes the implementation contract when deployed. The vulnerability was reported [here](https://forum.openzeppelin.com/t/uupsupgradeable-vulnerability-post-mortem/15680).